chance.js

```diff
@@ -133,6 +133,16 @@
         return this.random() * 100 < options.likelihood;
     };
 
+    Chance.prototype.animal = function (options){
+      //returns a random animal
+      options = initOptions(options);
+      if(typeof options.type !== 'undefined'){
+          return this.pick(this.get("animals")[options.type.toLowerCase()]);
```

Comment:
I think there should be a check here to ensure the type provided is one that we actually understand.

Your last commit basically had this by checking that it existed and throwing an error otherwise. I think the same should happen here so if I call it like `chance.animal({ type: 'banana' })` I get an error informing me that `'banana'` is not a recognized animal type rather than `undefined` which I would as this is written now.